src/migration.types.ts

```diff
@@ -174,10 +174,18 @@ export type MigrationFunction<
 
 /**
  * The behavior when an error occurs during the migration of an item.
- * Assuming that the error occurred at step p out of a total of n:
- * - "keep-original": the item is kept untouched.
- * - "keep-last-successful-version": the item obtained after the first p-1 successful steps is kept, and the n-p remaining steps are not applied.
- * - "keep-going": the n-p remaining steps are applied to the item obtained after step p, despite the error.
+ * This has an effect only when migrating through several versions in one go.
+ *
+ * For example, suppose that you're migrating from 5.0 to 5.3.
+ * For each saved item (e.g. a dashboard), three migration steps are applied: 5.0 => 5.1, 5.1 => 5.2, and 5.2 => 5.3.
+ * Each of these three steps might fail.
+ *
+ * More generically, assuming that the error occurred at step p out of a total of n, you can choose one of the following behaviors:
```

Comment:
Maybe now that you gave the example just above, it might be worth completing each case description by explaining what would happen if in the example above the fail happened at step 2?

Reply:
The example was just meant to explain that the option only has an impact if there are n >= 2 migration steps.
I think it's nice to keep the explanation generic after that.

Reply:
I meant keep the generic explanations, which I agree are nice, but complete each of them with a quick illustration of what they would give with in your example.

Reply:
I'm afraid that it would double the size of the description, which is already rather big.
So if you don't mind I would prefer to keep it this way for now, and we can always change it later if people struggle with it.

Completely unrelated, but I think most consumers will just use this package as a CLI and won't fork it, so they'll never see this JSDoc.
I think it's still nice to keep it internally, but the most important is to have this description in the `yargs` CLI.